cli/cmd/install.go

```diff
@@ -744,7 +747,7 @@ func (options *installOptions) globalConfig(identity *pb.IdentityContext) *pb.Gl
 		Version:                options.controlPlaneVersion,
 		IdentityContext:        identity,
 		OmitWebhookSideEffects: options.omitWebhookSideEffects,
-		ClusterDomain:          defaultClusterDomain,
```

Comment:
Is this variable still defined? It's not removed in this pull request and the default above pulls it from `defaults.ClusterDomain`. Can it be dropped or am I missing something?

Reply:
`defaultClusterDomain` is still used in the other cmd commands like generating service profiles when fetching the cluster domain from the cluster fails.

Those occasions can possibly also be replaced by `defaults.ClusterDomain`. I'll take a look at it.

Reply:
> Those occasions can possibly also be replaced by defaults.ClusterDomain. I'll take a look at it.

:thinking:  That'd mean we'd need to generate the default helm values from the charts in the other cmd commands that rely on `defaultClusterDomain` only to get `cluster.local`. Not sure that trade of is worth it, although it wouldn't be too much code:

```
if clusterDomain == "" {
  defaults, err := charts.NewValues(false)
  if err != nil {
    return err
  }
  clusterDomain = defaults.ClusterDomain
}
```

Reply:
I see. 😅